build_daemon/lib/client.dart

```diff
@@ -102,7 +102,8 @@ class BuildDaemonClient {
     var result = await stream.firstWhere(_isActionMessage, orElse: () => null);
 
     if (result == null) {
-      throw Exception('Unable to start build daemon: ${await output}');
+      throw Exception(
```

Comment:
Didn't notice this before - but in general we don't want to row a bare `Exception` - better to subclass it with something specific. We don't need to resolve in this PR though.